src/main/java/edu/ie3/datamodel/utils/validation/ConnectorValidationUtils.java

```diff
@@ -124,7 +124,7 @@ private static void checkTransformer2W(Transformer2WInput transformer2W) {
    * - rSc is greater 0 (short circuit resistance) <br>
    * - xSc is greater 0 (short circuit impedance) <br>
    * - gM is greater/equal to 0 (no load conductance) <br>
-   * - bM is greater/equal to 0 (no load susceptance) <br>
+   * - bM is less/equal to 0 <br>
```

Comment:
```suggestion
   * - bM is less/equal to 0 (no load susceptance)<br>
```

Please keep this part of the documentation